### DISCUSSION TITLE
Question 43: Mammals Embryo’s

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1040] [Q&A]
Description: Mammal’s aren’t documenting embryo’s in the way that Darwin Core indicates that it should be categorized. How should it be done correctly?

**Comment by rondlg** [comment]
Response: See Preps Parts conversation for further discussion. We will need shared workflows for this.

**Reply by rondlg** [reply]
@fmjjones Did this get discussed at Preps/Parts? If so can you drop a link to the discussion here please?

**Comment by fmjjones** [comment]
This was discussed at the last EMu users group and Mammals will follow DarwinCore standards by recording embryos in PheStage_tab field. See https://docs.google.com/spreadsheets/d/1Hpaxm16huQ7Lw3IMJKyOlJA4mhRpPL3s5Fg_rUFpsO4/edit#gid=1440398039 homework and list of approved preps.
